Dockerfile

```diff
@@ -12,8 +12,9 @@ ENV DB_PORT 5432
 ENV DB_NAME pega
 
 # Provide variables for the JDBC connection string
-ENV JDBC_URL "jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}"
 ENV JDBC_CLASS org.postgresql.Driver
+ENV JDBC_DB_TYPE postgresql
```

Comment:
Maybe remove `JDBC_DB_TYPE` as well and just parse it from the passed `JDBC_URL`? That's what we do here in our `gradle-prpc-plugin` and would remove one more variable to track.